server/config.ts

```diff
@@ -68,6 +68,15 @@ export default {
     expiryMinutes: Number(get('WEB_SESSION_TIMEOUT_IN_MINUTES', 120)),
   },
   apis: {
+    approvedPremises: {
```

Comment:
This has now been added in a [previous commit](https://github.com/ministryofjustice/hmpps-community-accommodation-tier-2-bail-ui/commit/91060d0d7cc5ed01b1dfe3a7383ede46b2461a50).